apps/src/templates/projects/PersonalProjectsTable.jsx

```diff
@@ -0,0 +1,301 @@
+import React, {PropTypes} from 'react';
+import i18n from "@cdo/locale";
+import color from "../../util/color";
+import FontAwesome from '@cdo/apps/templates/FontAwesome';
+import {ImageWithStatus} from '../ImageWithStatus';
+import {Table, sort} from 'reactabular';
+import wrappedSortable from '../tables/wrapped_sortable';
+import orderBy from 'lodash/orderBy';
+import {
+  personalProjectDataPropType,
+  PROJECT_TYPE_MAP
+} from './projectConstants';
+import QuickActionsCell from '../tables/QuickActionsCell';
+import {tableLayoutStyles, sortableOptions} from "../tables/tableConstants";
+import PopUpMenu from "@cdo/apps/lib/ui/PopUpMenu";
+
+const PROJECT_DEFAULT_IMAGE = '/blockly/media/projects/project_default.png';
+
+const THUMBNAIL_SIZE = 65;
+
+/** @enum {number} */
+export const COLUMNS = {
+  THUMBNAIL: 0,
+  PROJECT_NAME: 1,
+  APP_TYPE: 2,
+  LAST_PUBLISHED: 3,
+  LAST_FEATURED: 4,
+  ACTIONS: 5,
+};
+
+export const styles = {
+  cellFirst: {
+    borderWidth: '1px 0px 1px 1px',
+    borderColor: color.border_light_gray,
+  },
+  headerCellFirst: {
+    borderWidth: '0px 0px 1px 0px',
+    borderColor: color.border_light_gray,
+  },
+  cellThumbnail: {
+    width: THUMBNAIL_SIZE,
+    minWidth: THUMBNAIL_SIZE,
+    padding: 2
+  },
+  headerCellThumbnail: {
+    padding: 0
+  },
+  cellName: {
+    borderWidth: '1px 1px 1px 0px',
+    borderColor: color.border_light_gray,
+    padding: 15,
+    width: 250
+  },
+  headerCellName: {
+    borderWidth: '0px 1px 1px 0px',
+    borderColor: color.border_light_gray,
+    padding: 15
+  },
+  cellType: {
+    width: 120
+  },
+  centeredCell: {
+    textAlign: 'center'
+  },
+  thumbnailWrapper: {
+    height: THUMBNAIL_SIZE,
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+  }
+};
+
+// Cell formatters.
+const thumbnailFormatter = function (thumbnailUrl, {rowData}) {
+  const projectUrl = `/projects/${rowData.type}/${rowData.channel}/`;
+  thumbnailUrl = thumbnailUrl || PROJECT_DEFAULT_IMAGE;
+  return (
+    <a style={tableLayoutStyles.link} href={projectUrl} target="_blank">
+      <ImageWithStatus
+        src={thumbnailUrl}
+        width={THUMBNAIL_SIZE}
+        wrapperStyle={styles.thumbnailWrapper}
+      />
+    </a>
+  );
+};
+
+const nameFormatter = (projectName, {rowData}) => {
+  const url = `/projects/${rowData.type}/${rowData.channel}/`;
+  return <a style={tableLayoutStyles.link} href={url} target="_blank">{projectName}</a>;
+};
+
+const actionsFormatter = (actions, {rowData}) => {
+  return (
+    <QuickActionsCell>
+      <PopUpMenu.Item
+        onClick={() => console.log("Rename was clicked")}
```

Comment:
as a next step, you can hook up callbacks to these click handlers, and then use storybook actions to show that these are getting clicked. for example: https://github.com/code-dot-org/code-dot-org/blob/4fe563bd712fae37d57d248dfd18f2c474328d40/apps/src/templates/publishDialog/PublishDialog.story.jsx#L21